scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -82,8 +82,20 @@
       <div>
         <strong>Local (arch.)</strong>: {{ variant.local_obs_old or 0 }} obs.
       </div>
+      {% if variant.cosmic_ids %}
+      <strong>CosmicIDs</strong>:
```

Comment:
Lets use count and change this to:
```suggestion
      <strong>COSMIC obs. </strong>:
```